pages/_app.tsx

```diff
@@ -62,6 +64,12 @@ type ComponentWithPageLayout = AppProps & {
 };
 
 function MyApp({ Component, pageProps }: ComponentWithPageLayout) {
+  useEffect(() => {
+    if (process.env.NEXT_PUBLIC_FARO_COLLECTOR_URL && process.env.NEXT_PUBLIC_FARO_APP_ENVIRONMENT != "local") {
+      initGrafanaFaro();
+    }
+  }, []);
+
```

Comment:
I'm unsure this is doing what I expect it to do: Faro needs to initialize on the client (not the server) so I figured that a `userEffect` would get the app to 

But it only seems to capture traces/profiles when the App first loads on the `/feed` and I see this log go through:

```
Faro
 Faro is already registered. Either add instrumentations, transports etc. to the global faro instance or use the "isolate" property
```

which makes me think that maybe there's a different way to instantiate it - Investigating.

Reply:
Ended up going with a `useRef` which now doesn't seem to cause Faro to be re-initilized

Reply:
What do the Faro docs recommend for a React app?